### DISCUSSION TITLE
I have the old FPDF installed, do I have to remove it before installing FPDF2?

### DISCUSSION THREAD

**Discussion by entzyeung** [py-pdf/fpdf2#408] [General]
I have the old FPDF installed, do I have to remove it before installing FPDF2? I am on python 3.8.

**Comment by Lucas-C** [comment]
Assuming you know how tu use a terminal:

1. (optional) use `pip freeze` to list the Python packages you have installed, with their versions
2. Use `pip uninstall fpdf` to remove PyFPDF
3. Use `pip install fpdf2` to install this library

**Reply by entzyeung** [reply]
thank you  !
